app/src/lib/components/HunkLines.svelte

```diff
@@ -118,7 +135,7 @@
 	.selectable-wrapper {
 		cursor: text;
 		display: inline-block;
-		text-indent: 4px;
-		margin-right: 4px;
+		text-indent: var(--size-4);
+		margin-right: var(--size-4);
```

Comment:
I think @PavelLaptev has phased out --size now in favour of automatically translating px into rem when building the app.

Reply:
Cleaned up :+1: